src/apps/frontend/types/account.ts

```diff
@@ -2,12 +2,14 @@ import { JsonObject } from './common-types';
 
 export class Account {
   id: string;
+  displayName: string;
```

Comment:
Instead of storing in property and computing in constructor, consider exposing `displayName = () => {}` method on the class.